### DISCUSSION TITLE
(NOT ISSUE!!!) When we gonna get an update??

### DISCUSSION THREAD

**Discussion by Sepotato5123** [Okabintaro/SubmersedVR#193] [Q&A]
We didn't get an update for almost 2 years, are we ever gonna get an update? I saw that Okabintaro updated the description.
(sorry for posting this in issues but I am new to github and I don't know where to leave comments)

**Comment by farmboy0** [comment]
Why do you want an update? Is there something missing or buggy make an issue for it.

**Comment by Sepotato5123** [comment]
As i've heard from other players, its a bit buggy at vehicles, and maybe some overhaul or improvements at the player swimming, I really see some more potential for the mod which that why I am asking if we gonna get some improvements/additions to it, and maybe more compatibility to other mods. 
Edit: and lets not forget the milestones that are not completed
